modules/idWardRtdProvider.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * This module adds the ID Ward RTD provider to the real time data module
+ * The {@link module:modules/realTimeData} module is required
+ * The module will poulate real-time data from ID Ward
+ * @module modules/idWardRtdProvider
+ * @requires module:modules/realTimeData
+ */
+import {config} from '../src/config.js';
+import {getStorageManager} from '../src/storageManager.js';
+import {submodule} from '../src/hook.js';
+import {isPlainObject, mergeDeep, logMessage, logError} from '../src/utils.js';
+
+const MODULE_NAME = 'realTimeData';
+const SUBMODULE_NAME = 'idWard';
+
+export const storage = getStorageManager(null, SUBMODULE_NAME);
+
+/**
+  * Lazy merge objects.
+  * @param {String} target
+  * @param {String} source
+  */
+function mergeLazy(target, source) {
```

Comment:
did you see the utils function that does this?

Reply:
used `mergeDeep`